packaging/source/buildpackage.sh

```diff
@@ -26,4 +28,6 @@ rm "${OUTPUTDIR}/OpenRA-${TAG}-source.tar" || :
 git ls-tree HEAD --name-only -r -z | xargs -0 tar vrf "${OUTPUTDIR}/OpenRA-${TAG}-source.tar"
 bzip2 "${OUTPUTDIR}/OpenRA-${TAG}-source.tar"
 
+make version VERSION='{DEV_VERSION}'
```

Comment:
We use these scripts for release builds so this is invalid.

Reply:
this doesn't affect the output, which goes to a different directory. It affects the file left in the source tree, which shouldn;t change during a release.

Reply:
You should use the Makefile directly instead of through these release scripts.